### ISSUE TITLE
c_cpp_properties.json: browse property path recursive search not working

### ISSUE THREAD

**Issue by DeadRabbits307** [microsoft/vscode-cpptools#4443]
**Type: LanguageService**
As described [here](https://code.visualstudio.com/docs/cpp/c-cpp-properties-schema-reference) an entry in browse/path like `${buildroot_root_dir}output/build/linux-4.15.16/drivers` shall recursive search all subdirectories of `drivers`. But this is not the case. I have to add each subdir to get to the definition of an function like:

```
"env": {
        "buildroot_root_dir": "/home/xy/buildroot/buildroot-2018.02.5/"
    },
    "configurations": [
        {
            "name": "Linux",
            "includePath": [
                "${workspaceFolder}/**"
            ],
            "compilerPath": "${buildroot_root_dir}output/host/usr/bin/aarch64-linux-gcc",
            "cStandard": "c11",
            "intelliSenseMode": "gcc-x64",
            "forcedInclude": [
                "${buildroot_root_dir}output/build/linux-4.15.16/include/generated/autoconf.h"
            ],
            "browse": {
                "path": [
                    "${workspaceFolder}/../../my_second_proj",
                    "${buildroot_root_dir}output/build/linux-4.15.16/drivers/base",
                    "${buildroot_root_dir}output/build/linux-4.15.16/drivers/crypto",
                    ...],
                ],
                "limitSymbolsToIncludedHeaders": false,
                "databaseFilename": ""
            },
            "compileCommands": "${workspaceFolder}/compile_commands.json"
      }
```
Only
```
"browse": {
                "path": [
                    "${workspaceFolder}/../../my_second_proj",
                    "${buildroot_root_dir}output/build/linux-4.15.16/drivers"
                  ]
...
```
does not enable the "Got to Definition" to find functions in                     `${buildroot_root_dir}output/build/linux-4.15.16/drivers/base/platform.c` for example.

**Describe the bug**
- OS and Version: 4.19.45-1-MANJARO
- VS Code Version: 1.34.0
- C/C++ Extension Version: 0.24.0

**Expected behavior**
```
"browse": {
                "path": [
                    "${workspaceFolder}/../../my_second_proj",
                    "${buildroot_root_dir}output/build/linux-4.15.16/drivers",
...
```
should be enough to find `${buildroot_root_dir}output/build/linux-4.15.16/drivers/base/platform.c` and any other file in` /drivers` or in any subdir in` /drivers ` with the "Got to Definition". 

Bug? Configuration issue?

**Comment by sean-mcmanus** [comment]
Are you using 0.24.0? Does it repro after you update to 0.26.0?

**Comment by DeadRabbits307** [comment]
I updated  to 0.26.0, this fixed it! (working in an offline environment)

In addition: the OUTLINE now works again. With 0.24.0 the "Follow Cursor" functionality did not work. I don't know if this is related to vscode-cpptools...

Thanks anyway!

**Comment by bobbrow** [comment]
Thanks for confirming!

And yes, we fixed the Document Outline in 0.26.0.
